### ISSUE TITLE
On beautify, new line before next CSS selector

### ISSUE THREAD

**Issue by djayard** [beautify-web/js-beautify#1142]
# Description
On every beautify operation, a new line is being injected into an html file. In other words, after 6 beautify runs, there's six new lines in the file. It's a work file, so I can't post the file's entire contents, but for some context, it's an AngularJS view and doesn't start with body, doctype, or html tags.

This occurs with both js-beautify and gulp-jsbeautifier.

js-beautify v4.1.2
gulp-jsbeautifier v2.0.4

# Input
The code looked like this before beautification:
```
<style>
    .checkbox input[type="checkbox"]:disabled+label::before {
        cursor: default !important;
    }

    ul.ui-autocomplete {
        display: block;
        top: 336px;
        left: 843.484px;
        z-index: 99999 !important;
        width: 354px;
    }
</style>
```
# Actual Output
The  code actually looks like this after 5 runs:
```
<style>
    .checkbox input[type="checkbox"]:disabled+label::before {
        cursor: default !important;
    }






    ul.ui-autocomplete {
        display: block;
        top: 336px;
        left: 843.484px;
        z-index: 99999 !important;
        width: 354px;
    }
</style>
```

# Steps to Reproduce
Example:
```shell
js-beautify -r --config beautify-settings.json view.html
```

## Environment
OS: Windows 10

## Versions
js-beautify v4.1.2
gulp-jsbeautifier v2.0.4

## Settings
Example:
```json
{
    "indent_size": 4,
    "indent_char": " ",
    "eol": "\r\n",
    "indent_level": 0,
    "indent_with_tabs": false,
    "preserve_newlines": true,
    "max_preserve_newlines": 10,
    "jslint_happy": false,
    "space_after_anon_function": false,
    "brace_style": "collapse",
    "keep_array_indentation": false,
    "keep_function_indentation": false,
    "space_before_conditional": true,
    "break_chained_methods": false,
    "eval_code": false,
    "unescape_strings": false,
    "wrap_line_length": 0,
    "wrap_attributes": "auto",
    "wrap_attributes_indent_size": 4,
    "end_with_newline": false
}
```

## Work Around
I seperated the CSS selectors:
```
<style>
    .checkbox input[type="checkbox"]:disabled+label::before {
        cursor: default !important;
    }
</style>
<style>
    ul.ui-autocomplete {
        display: block;
        top: 336px;
        left: 843.484px;
        z-index: 99999 !important;
        width: 354px;
    }
</style>
```

**Comment by bitwiseman** [comment]
There is no `js-beautify v4.1.2`, but this is a but introduced in 1.6.10.

**Comment by bitwiseman** [comment]
Fixed by rolling back #1117.

**Comment by ram-you** [comment]
Sorry for commenting on a closed issue.
Regression: The same error occurred with the new version 0.8.9
